api/src/main/java/org/ehrbase/api/service/CompositionService.java

```diff
@@ -172,4 +173,6 @@
     Integer getVersionByTimestamp(UUID compositionId, LocalDateTime timestamp);
 
     UUID getSystemUuid();    // from BaseService
+
+    ServerConfig getServerConfig(); // from BaseService
```

Comment:
I may be time to have an Interface BaseService and have CompositionService....  extend that.